src/components/CurrentWalletAdvanced.test.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+// @ts-ignore
```

Comment:
What does that do / why is it needed? 🤔

Reply:
It is needed in the transition while we support js and ts.
If the line is not ignored by ts, the following warning is emitted:
> `Could not find a declaration file for module '../testUtils'. 'joinmarket-webui/src/testUtils.jsx' implicitly has an 'any' type.ts(7016)`

Either ignore the line with `// @ts-ignore` temporarily or disable `noImplicitAny` compiler options.
I wanted to go with the former, as removing the option seems to induce more work later on.

So it is just needed when importing code via object destructuring from js in a ts file. 
Do you think we should leave it that way or is there maybe an other, better solution to this?

Reply:
Ok I see. Let's leave it like this! ✅